packages/common/src/store/purchase-content/sagas.ts

```diff
@@ -1160,6 +1155,17 @@ function* watchStartPurchaseContentFlow() {
   yield takeLatest(startPurchaseContentFlow, doStartPurchaseContentFlow)
 }
 
+function* watchEnsureUserBankExists() {
+  yield takeEvery(ensureUserBankExists, function* () {
+    try {
+      yield* call(getOrCreateUSDCUserBank)
+    } catch (e) {
+      // No need to bubble here as later flows will still attempt to create it
+      console.warn(`ensureUserBankExists failed: ${e}`)
```

Comment:
nit: what if future callers don't necessarily attempt to create the user bank afterwards? another idea is to console.error so it gets to sentry at least?

Reply:
Fair point. This is only being called in the purchase flows at the moment. And those _definitely_ also call `getOrCreateUSDCUserBank`. I was trying to avoid having multiple errors bubble up for the same reason. But it's probably fine.